src/ReactWindowGrid.js

```diff
@@ -5,13 +5,13 @@ import scrollbarSize from 'dom-helpers/scrollbarSize'
 
 const absolute = 'absolute'
 
-const getText = value => String(value === undefined ? '' : value)
+const getText = (value) => String(value === undefined ? '' : value)
```

Comment:
Avoid unnecessary changes following the previous code pattern, in that case do not use parentheses and bellow not adding trailing commas

Reply:
All formatting have been corrected as of commit 875d7b1